[stable/mongodb] Add priority class to pods

viktor42b opened the pull request:
#### What this PR does / why we need it:

This PR allows user to specify PriorityClass for mongodb pods

#### Checklist

- [X] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md

tompizmor committed:
[stable/rabbitmq] Refactor way to add custom configuration (#10070)

* [stable/rabbitmq] Remove bad config option from values.yaml

Signed-off-by: tompizmor <tompizmor@gmail.com>

* Delete from both values files

Signed-off-by: tompizmor <tompizmor@gmail.com>

* Split config map content in two configuration values

Signed-off-by: tompizmor <tompizmor@gmail.com>

* Apply feedback

Signed-off-by: tompizmor <tompizmor@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

tschmittni committed:
[stable/redis-ha] Fix for using existing secret value (#10693)

The PR https://github.com/helm/charts/pull/10032 injects the AUTH
secret into the redis and sentinel containers but does not
consider the existingSecret parameter.

Signed-off-by: Thomas Schmitt <thomas.schmitt@ni.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
Add priorityClassName to mongodb statefulset and deployment

Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

lewiscowper committed:
Fix typo (#10694)

Signed-off-by: Lewis Cowper <lewis.cowper@googlemail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

pawelbojanowski committed:
[stable/sumologic-fluentd] Control adding time and stream via values (#9095)

Signed-off-by: pawelbojanowski <pawel.bojanowski@saucelabs.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

robinelfrink committed:
[stable/nfs-client-provisioner] Fix deployment to use resources (helm… (#10702)

* [stable/nfs-client-provisioner] Fix deployment to use resources (helm#10701).

Signed-off-by: Robin Elfrink <robin.elfrink@eu.equinix.com>

* [stable/nfs-client-provisioner] Bump Chart version.

Signed-off-by: Robin Elfrink <robin.elfrink@eu.equinix.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
odoo: update to `11.0.20190115` (#10700)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

ephur committed:
incorrect value for readiness probe path (#10708)

Signed-off-by: ephur <richard.maynard@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

varshavaradarajan committed:
[stable/gocd] Bump up artifact plugin to latest version (#10709)

* [stable/gocd] Bump up the version of docker registry artifact plugin.

Signed-off-by: Varsha Varadarajan <varshasvaradarajan@gmail.com>

* [stable/gocd] Update changelog

Signed-off-by: Varsha Varadarajan <varshasvaradarajan@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

mumoshu committed:
[stable/envoy]: optional feature to generate envoy.yaml from golang template (#10265)

This feature enables you to make your values.yaml a bit more DRY by referencing chart values from within the envoy.yaml template by using the `{{ .Values.foo.bar }}`.

This also adds mumoshu as an co-owner of this chart.

Signed-off-by: Yusuke KUOKA <ykuoka@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
[stable/spinnaker] specify which namespaces to omit (#10029)

* can specify which k8s namespaces to omit

Signed-off-by: David O'Dell <dave@helix.re>

* bumped chart version to 1.1.7

Signed-off-by: David O'Dell <dave@helix.re>

* fixed lint error

Signed-off-by: David O'Dell <dave@helix.re>

*  passing lint test now

Signed-off-by: David O'Dell <dave@helix.re>

* [stable/spinnaker] Version bump

Signed-off-by: Vic Iglesias <viglesias@google.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

verwilst committed:
Fix chart icon (#10703)

Signed-off-by: Bart Verwilst <bart@verwilst.be>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Place1 committed:
grafana will now tpl datasources (#10603)

* grafana will now tpl datasources

Signed-off-by: James Batt <getjamesbatt@gmail.com>

* updated docs

Signed-off-by: James Batt <getjamesbatt@gmail.com>

* version bump

Signed-off-by: James Batt <getjamesbatt@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
[stable/sumologic-fluentd] Fix render error if daemonset.priorityClassName not specified (#10698)

Signed-off-by: Anton Petrov <petrov.aap@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

tompizmor commented on stable/mongodb/Chart.yaml:
```diff
@@ -1,5 +1,5 @@
 name: mongodb
-version: 5.1.3
+version: 5.1.4
```
As you are including a new feature, you should update the minor version.

```suggestion
version: 5.3.0
```

> I propose the version 5.3.0 because 5.2.0 already exists in master after https://github.com/helm/charts/pull/10721 was merged

viktor42b replied:
done!

Commit:
phabricator: update to `2019.2.0` (#10718)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

tompizmor commented on stable/mongodb/values-production.yaml:
```diff
@@ -138,6 +138,8 @@ resources: {}
 #   cpu: 100m
 #   memory: 256Mi
 
+# priorityClassName: ""
```
Can you add a comment before the parameter? Something like:

```
## Pod priority
## https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
# priorityClassName: ""
```

viktor42b replied:
done!

tompizmor reviewed (changes requested):
Thanks for the PR! Please take a look to my comments

Commit:
ghost: update to `2.11.1` (#10697)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
drupal: update to `8.6.7` (#10720)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

tompizmor committed:
[stable/mongodb] Add option to disable mongodb system log (#10721)

Signed-off-by: tompizmor <tompizmor@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

thomasdarimont committed:
[stable/keycloak] Update to 4.8.3 and related changes (#10713)

* [stable/keycloak] Upgrade Keycloak to 4.8.3.Final and related changes

Added support for more reliable database connection handling.

Referenced missing distributed caches in ha.cli configuration.

Configured events-listener SPI to log login events to stdout for analysis

Signed-off-by: Thomas Darimont <thomas.darimont@gmail.com>

* Add line break at end of file

Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

* Update README.md

Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

plumdog committed:
[stable/ark] CRD hooks and schedules (#7165)

* [stable/ark] Refactor to use crd-install hook

Signed-off-by: Andrew Plummer <plummer574@gmail.com>

* [stable/ark] Add option to create schedules

Signed-off-by: Andrew Plummer <plummer574@gmail.com>

* [stable/ark] Bump chart version number

Signed-off-by: Andrew Plummer <plummer574@gmail.com>

* [stable/ark] Set required tiller version

Signed-off-by: Andrew Plummer <plummer574@gmail.com>

* [stable/ark] CR: Un-capture root context

Signed-off-by: Andrew Plummer <plummer574@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

nitisht committed:
Fix liveness probes and update Minio version (#10704)

Signed-off-by: Nitish Tiwari <nitish@minio.io>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

paulczar committed:
[stable/spinnaker] add external secret support for GCS (#10716)

* [stable/spinnaker] add external secret support for GCS

Allow user to supply their own pre-created secret for GCS storage

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

* fix faulty if statements

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

paskal committed:
[stable/prometheus-operator] sync prometheus rules from coreos and etcd upstream (#10538)

* [stable/prometheus-operator] move rules and dashboards to new locations

Signed-off-by: Dmitry Verkhoturov <paskal.07@gmail.com>

* [stable/prometheus-operator] sync rules with upstream

Signed-off-by: Dmitry Verkhoturov <paskal.07@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

ivanmp91 committed:
Fix/prometheus adapter image pull secrets (#10725)

* Move image pull secret block to a proper place

Signed-off-by: Iván Mora <ivan.mora@ST-A135-5LLG8WP.domino.softonic.com>

* Bump chart version

Signed-off-by: Iván Mora <ivan.mora@ST-A135-5LLG8WP.domino.softonic.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
[stable/mariadb] Allow setting the 'serviceAccountName' (#10726)

* [stable/mariadb] Allow setting the 'serviceAccountName'

Signed-off-by: juan131 <juan@bitnami.com>

* Bump minor version

Signed-off-by: juan131 <juan@bitnami.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

kimxogus committed:
[stable/elasticsearch-curator] add jobs in hooks, delete .Values.config (#10049)

* deprecate .Values.config

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

* add hooks

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

idanlevin committed:
Fix pushgateway labels honoring (#10728)

* add honorLabels to Pushgateway ServiceMonitor

Signed-off-by: Idan Levin <idlevin@microsoft.com>

* Pushgateway chart version bump

Signed-off-by: Idan Levin <idlevin@microsoft.com>

* update readme

Signed-off-by: Idan Levin <idlevin@microsoft.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

aeijdenberg committed:
Stop using POD IP for liveness checks so that it works with ISTIO mTLS (#10689)

If mutual TLS is enabled in strict mode with ISTIO, then the Postgresql
liveness and readiness checks fail to succeed, as traffic to pod IPs is
blocked by the sidecar proxy.

See: https://istio.io/help/faq/security/#k8s-health-checks

Signed-off-by: Adam Eijdenberg <adam.eijdenberg@digital.gov.au>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

ephur committed:
update default to latest Karma release (#10736)

Signed-off-by: ephur <richard.maynard@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

jhohertz committed:
This adds the aggregated-metrics-reader ClusterRole which was missing. (#10737)

See deploy manifest in the metrics-server project here:
https://github.com/kubernetes-incubator/metrics-server/blob/master/deploy/1.8%2B/aggregated-metrics-reader.yaml

Signed-off-by: Joe Hohertz <joe@viafoura.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

plumdog committed:
Cluster autoscaler clusterrole add jobs (#10206)

* [cluster-autoscaler] Add batch.jobs permissions to clusterrole

Signed-off-by: Andrew Plummer <plummer574@gmail.com>

* [cluster-autoscaler] Cronjobs too

Signed-off-by: Andrew Plummer <plummer574@gmail.com>

* [cluster-autoscaler] Bump version number

Signed-off-by: Andrew Plummer <plummer574@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

omerxx committed:
Adding an option to use memoryMode for ETCD  (#10436)

* Adding an option to use memoryMode for ETCD when persistentVolumes are not enabled

Signed-off-by: omerxx <omerxx@gmail.com>

* Version bump to v0.6.1, documentation and README entry

Signed-off-by: omerxx <omerxx@gmail.com>

* Remove trailing spaces

Signed-off-by: omerxx <omerxx@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

glebsam committed:
added missed switcher for RBAC-disabled deployment (#9484)

* added missed switcher for RBAC-disabled deployment

Signed-off-by: Gleb Samsonenko <ilot999@gmail.com>

* added blank line at the end of role.yaml

Signed-off-by: Gleb Samsonenko <ilot999@gmail.com>

* Updated chart version

Signed-off-by: Gleb Samsonenko <ilot999@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

paulczar committed:
[stable/gce-ingress] fix service selector for backend (#10735)

the service was selecting for the wrong pod.

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

davidkarlsen committed:
get checksum of configmap (#10753)

Signed-off-by: David J. M. Karlsen <david@davidkarlsen.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

kimxogus committed:
fix fullname (#10752)

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

IvanovOleg committed:
Fixed the broker id (#10591)

Signed-off-by: IvanovOleg <olivanov87@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

nreisbeck committed:
stable/traefik/Chart.yaml: Update Traefik image to v1.7.7 (#10748)

* stable/traefik/Chart.yaml: Update Traefik image to v1.7.7

Signed-off-by: Nolan Reisbeck <nolan.reisbeck@gmail.com>

* update readme too

Signed-off-by: David J. M. Karlsen <david@davidkarlsen.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

goddenrich committed:
[stable/concourse] Specify secrets mount path (#9364)

* control the mount paths of the secret keys and certs

Signed-off-by: Richard Godden <goddenrich@gmail.com>

* added variables to readme

Signed-off-by: Richard Godden <goddenrich@gmail.com>

* secrets mountpath now has defaults

Signed-off-by: Richard Godden <goddenrich@gmail.com>

* using web and worker keys to specify secrets mount path

Signed-off-by: Richard Godden <goddenrich@gmail.com>

* lint

Signed-off-by: Richard Godden <goddenrich@gmail.com>

* changed secrets path keys in README

Signed-off-by: Richard Godden <goddenrich@gmail.com>

* setting secret mount path defaults in the values.yaml instead of the templates

Signed-off-by: rgodden <rgodden@thoughtmachine.net>

* specifying mount path for tls

Signed-off-by: rgodden <rgodden@thoughtmachine.net>

* bump version

Signed-off-by: rgodden <rgodden@thoughtmachine.net>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

yuchaoran2011 committed:
Upgrade to beta API (#10772)

* Upgrade to beta API

Signed-off-by: Chaoran Yu <yuchaoran2011@gmail.com>

* Changed default job namespace

Signed-off-by: Chaoran Yu <yuchaoran2011@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

paulczar committed:
[stable/spinnaker] upgrade to latest version of spinnaker (#10717)

* [stable/spinnaker] upgrade to latest version of spinnaker

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

* Update Chart.yaml

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

* Update values.yaml

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
phabricator: update to `2019.3.0` (#10777)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

carrodher committed:
[stable/bookstack] Add png logo to be displayed in https://hub.helm.sh (#10260)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

nreisbeck committed:
stable/unifi/README.md: fix current version (#10784)

Signed-off-by: Nolan Reisbeck <nolan.reisbeck@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Miouge1 committed:
Add prometheus-snmp-exporter chart (#10734)

* Add prometheus-snmp-exporter chart

Signed-off-by: Maxime Guyot <maxime@root314.com>

* Add RBAC and service accounts best practices

Signed-off-by: Maxime Guyot <maxime@root314.com>

* Use the new labels

Signed-off-by: Maxime Guyot <maxime@root314.com>

* Add documentation of the Prometheus configuration

Signed-off-by: Maxime Guyot <maxime@root314.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

cpanato committed:
bump mm-te to 5.7.0 (#10778)

Signed-off-by: cpanato <ctadeu@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
redmine: update to `4.0.1` (#10787)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

mcfedr committed:
Fix ingress for GCE (#10745)

* Fix ingress for GCE

The GCE ingress controller fails when the ingress name contains dots.

Signed-off-by: Fred Cox <mcfedr@gmail.com>

* Fix version

Signed-off-by: Fred Cox <mcfedr@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

pytimer committed:
[incubator/gogs] Add custom nodePort configuration (#10792)

* [incubator/gogs] Add custom nodePort configuration

Signed-off-by: pytimer <lixin20101023@gmail.com>

* [incubator/gogs] bump version to 0.7.6

Signed-off-by: pytimer <lixin20101023@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Brightside56 committed:
[stable/kubernetes-dashboard] Add --enable-skip-login option/parameter to chart (#10802)

* Adding skip login option to deployment

Signed-off-by: Brightside56 <o.gumbar@papakaya.com>

* Add skip login value

Signed-off-by: Brightside56 <o.gumbar@papakaya.com>

* Add extra arg for skip login

Signed-off-by: Brightside56 <o.gumbar@papakaya.com>

* Add documentation for skip login

Signed-off-by: Brightside56 <o.gumbar@papakaya.com>

* Bump version

Signed-off-by: Brightside56 <o.gumbar@papakaya.com>

* Update deployment.yaml

Signed-off-by: Brightside56 <o.gumbar@papakaya.com>

* Update README.md

Signed-off-by: Brightside56 <o.gumbar@papakaya.com>

* Update values.yaml

Signed-off-by: Brightside56 <o.gumbar@papakaya.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

spuder committed:
Remove 'alpha' reference from mysql readme (#10213)

* Remove 'alpha' reference from mysql readme

Signed-off-by: Spencer Owen <owenspencer@gmail.com>

* Bump mysql version

Signed-off-by: Spencer Owen <owenspencer@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
[stable/hazelcast-jet] Hazelcast Jet Update README.md (#10798)

* Update README.md

Fix typos around repository name being `hazelcast` instead `stable`

Signed-off-by: Emin Demirci <emin@hazelcast.com>

* Bump chart version

Signed-off-by: Emin Demirci <emin@hazelcast.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

v-yarotsky committed:
[stable/concourse] Workaround for https://github.com/concourse/concourse#2191 (#10773)

This commit allows to use AWS Secrets Manager without simultaneously
activating AWS SSM, and vice versa.
This should not be a problem in concourse 5.0.0, but the 4.x.x is still
affected.

Signed-off-by: Vladimir Yarotsky <vladimir@pubnub.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

natebwangsut committed:
[stable/consul] Added an option to insert uiService.annotation (#10796)

* Added an option for consul-ui to specify extra annotations

Signed-off-by: natebwangsut <nate.bwangsut@gmail.com>

* Bump chart version

Signed-off-by: natebwangsut <nate.bwangsut@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

fabriziofortino committed:
[stable/schema-registry-ui] add support for service annotations, documented missing params in README (#10806)

Signed-off-by: Fabrizio Fortino <fabrizio.fortino@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

goddenrich committed:
Webservice labels (#10808)

* specify additional labels for web service

Signed-off-by: Richard Godden <goddenrich@gmail.com>

* added web service labels to README

Signed-off-by: Richard Godden <goddenrich@gmail.com>

* bump version

Signed-off-by: Richard Godden <goddenrich@gmail.com>

* bump version

Signed-off-by: David J. M. Karlsen <david@davidkarlsen.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

bobvawter committed:
cockroachdb: Update to 2.1.4 (#10810)

* cockroachdb: Update to 2.1.4

Signed-off-by: Bob Vawter <bob@cockroachlabs.com>

* Update Chart.yaml

Signed-off-by: David J. M. Karlsen <david@davidkarlsen.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

valdisrigdon committed:
Runs tpl on dashboardsConfigMaps (#10670)

Signed-off-by: Valdis Rigdon <valdis.rigdon@appian.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

belminf committed:
[stable/nginx-ingress] Modifies PSP to allow for hostNetwork if enabled for controller (#10776)

* Modifies PSP to allow for hostNetwork if enabled for controller

Signed-off-by: Belmin Fernandez <belminf@gmail.com>

* Bumps chart version

Signed-off-by: Belmin Fernandez <belminf@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

kimxogus committed:
[stable/stolon] external secret, auto cluster spec update, pdb (#9739)

* add support for external stolon secret

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

* fix version

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

* fix rbac

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

* fix Chart.yaml

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

* add pdb

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

* update README.md

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

* auto update cluster spec

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

* disable pdb by default

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

* add failKeeper hook

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

* add sentinel pod checksum annotation

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

* remove podDisruptionBudget.enabled

Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

pulledtim committed:
Mysqldump service account injection (#10719)

* [mysqldump] Add support for the gcp service account controller

Signed-off-by: tim.smyth <tim.smyth@kiwigrid.com>

* [mysqldump] Use the custom filename in secret

Signed-off-by: tim.smyth <tim.smyth@kiwigrid.com>

* [mysqldump] Add labels

Signed-off-by: tim.smyth <tim.smyth@kiwigrid.com>

* [mysqldump] Don't create secret if existing secret specified

Signed-off-by: tim.smyth <tim.smyth@kiwigrid.com>

* [mysqldump] Reduced permission set

Signed-off-by: tim.smyth <tim.smyth@kiwigrid.com>

* [mysqldump] Added missing space in comments

Signed-off-by: tim.smyth <tim.smyth@kiwigrid.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
Adding chart for cluster-autoscaler overprovisioning (#10100)

* # This is a combination of 2 commits.
# This is the 1st commit message:

Initial commit

Signed-off-by: Max Williams <max.williams@deliveryhero.com>

# This is the commit message #2:

updating sources

* Commit Sign-off

Signed-off-by: Miguel Angel Mingorance Fernandez <miguel.mingorance@deliveryhero.com>

* Updating README table

Signed-off-by: Miguel Angel Mingorance Fernandez <miguel.mingorance@deliveryhero.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Simwar committed:
Datadog helm update (#10827)

* updated comment about useHostPort for APM

Signed-off-by: Simon Guerrier <simon.guerrier@datadoghq.com>

* changed liveness probe and added env var DD_HEALTH_PORT

Signed-off-by: Simon Guerrier <simon.guerrier@datadoghq.com>

* bump version to 1.17.0

Signed-off-by: Simon Guerrier <simon.guerrier@datadoghq.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

linki committed:
[stable/chaoskube]: update to v0.12.1, add logo and metadata (#10822)

Signed-off-by: Martin Linkhorst <martin.linkhorst+github.com@posteo.de>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

lachie83 committed:
update approvers based on lastest maintainership (#10831)

Signed-off-by: Lachlan Evenson <lachlan.evenson@microsoft.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
mediawiki: update to `1.32.0` (#10819)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
ghost: update to `2.12.0` (#10820)

Signed-off-by: Bitnami Containers <containers@bitnami.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

joshmyers committed:
[stable/efs-provisioner] Allow passing in annotations via values (#10829)

This commit enables passing in annotations via values for the
`efs-provisioner` chart and bumps the version to 0.1.3

Signed-off-by: Josh Myers <joshuajmyers@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

lachie83 committed:
Add emeritus section to OWNERS (#10832)

Signed-off-by: Lachlan Evenson <lachlan.evenson@microsoft.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

swade1987 committed:
Adding affinity options to KIAM agent and server deployments. (#10815)

* Adding affinity options to KIAM agent and server deployments.

Signed-off-by: Steven Wade <steven@stevenwade.co.uk>

* Bumping the KIAM chart version.

Signed-off-by: Steven Wade <steven@stevenwade.co.uk>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

bendrucker committed:
sumologic: add "extraEnv" for specifying custom env vars (#10837)

Signed-off-by: Ben Drucker <bvdrucker@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

billimek committed:
use a default values for the `flows` setting (#10765)

details: if the `flows` setting is left unset, node-red will default the
files it creates for new flows to match the hostname of the container
it's running under. This hostname is dynamic and changes with a pod
restart. As a result changes made and persisted will be lost.

By defaulting the `flows` value to some static value, we ensure that the
changes will persist between pod restarts (assuming using persistent
storage)

Signed-off-by: Jeff Billimek <jeff@billimek.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

kppullin committed:
EFS - Grant 'endpoints' access (#9127)

* EFS ClusterRole - Grant access to 'endpoint' resources

Signed-off-by: Kevin Pullin <kevin.pullin@gmail.com>

* Do not create EFS deployment if default EFS id set

Signed-off-by: Kevin Pullin <kevin.pullin@gmail.com>

* Bump efs-provisioner chart version

Signed-off-by: Kevin Pullin <kevin.pullin@gmail.com>

* Fix indentation from github UI conflict edit

Signed-off-by: Kevin Pullin <kevin.pullin@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

kfox1111 committed:
Add a kubectl proxy to allow rego to make kubernetes sars (#10490)

* Add a kubectl proxy to allow rego to make kubernetes subject access reviews.

Signed-off-by: Kevin Fox <kevin@efox.cc>

* Bumped the wrong version.

Signed-off-by: Kevin Fox <kevin@efox.cc>

* Fix lint.

Signed-off-by: Kevin Fox <kevin@efox.cc>

* Update based on feedback.

Update based on feedback by tsandall and timothyhinrichs.
Added OWNERS too for ease of management.

Signed-off-by: Kevin Fox <kevin@efox.cc>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

DizTortion committed:
[stable/docker-registry] add configuration option for S3 region endpoint (#10201)

* add configuration option for S3 region endpoint

Signed-off-by: Stefan Hoffmeister <stefan@hoffmeister.biz>

* bump chart version

Signed-off-by: Stefan Hoffmeister <stefan@hoffmeister.biz>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

aerotog committed:
Fix fatal error caused by Postgresql probes (#10838) (#10839)

Signed-off-by: Timothy Guenthner <aerotog@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

cesaralbloz committed:
[stable/grafana] Add gerrit server as dashboard origin (#10711)

* [stable/grafana] Add gerrit server as dashboard origin

Signed-off-by: cesaralbloz <cesaralbloz@hotmail.com>

* [stable/grafana] Add gerrit server as dashboard origin

Signed-off-by: Cesar Alba Lozano <cesaralbloz@hotmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

kaarolch committed:
[incubator/kafka] Change internal advertise lisiner to pod name instead IP (#10817)

Signed-off-by: Karol Chrapek <kchrapek@novomatic-tech.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

unguiculus committed:
[ci] Upgrade to chart-testing v2.2.0 (#10843)

Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

gregdhill committed:
[stable/helm] initialize chart in helm repo (#10367)

* hoard chart init

Signed-off-by: greg <greg.hill@monax.io>

* @cpanato comments

Signed-off-by: greg <greg.hill@monax.io>

* fix indentations

Signed-off-by: greg <greg.hill@monax.io>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

paulczar committed:
[stable/drone] Update drone to support v1 release candidate (#10614)

* Update drone to support v1 release candidate

This is a set of breaking changes that brings drone v1 support to the
chart. drone v1 itself is a breaking change and brings along native
kubernetes support.

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

* update chart app version to be accurate

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

* add newlines to the end of files

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

* remove $ from commands (markdown lint)

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

munnerz committed:
cert-manager: fast-forward to upstream 9a8f4371 (#10830)

* Bump version strings for v0.6.0 (jetstack/cert-manager#1247)
* Set CA sync backoff limit & add verifying installation docs (jetstack/cert-manager#1245)
* Added securityContext to cert-manager deployment pods in helm chart (jetstack/cert-manager#1237)
* Move Helm chart to deploy/charts/cert-manager (jetstack/cert-manager#1235)
* chart: allow deployment strategy customization
* webhook: remove hardcoded image (jetstack/cert-manager#1039)
* Remove crd-install annotation from Challenges and Orders
* Add Challenge and Order CRDs
* Add certificates and issuers to aggregated RBAC for view, edit and admin ClusterRoles. (jetstack/cert-manager#872)
* Use fixed image tag for ca-helper and add description

Signed-off-by: James Munnelly <james@munnelly.eu>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

smatyas committed:
[stable/traefik] dashboard serviceType (#10788)

* added dashboard.serviceType value to be able to deploy to GKE

Signed-off-by: Ole Rößner <o.roessner@neusta.de>

* added dashboard.serviceType value to be able to deploy to GKE

Signed-off-by: Ole Rößner <o.roessner@neusta.de>

* added dashboard.serviceType value to be able to deploy to GKE

Signed-off-by: Ole Rößner <o.roessner@neusta.de>

* removed executable flag from yaml files

Signed-off-by: Matyas Somfai <somfai.matyas@gmail.com>

* added a default value for dashboard.serviceType; modified the service to use a named port; fixed a link in the readme

Signed-off-by: Matyas Somfai <somfai.matyas@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

gezb committed:
[Stable/chartmuseum] Allow external secrets (#10818)

* [stable][chartmuseum] Allow the use of external secrets

Signed-off-by: Gerald Barker <gerald.barker@gmail.com>

* Update Table Of Content

Signed-off-by: Gerald Barker <gerald.barker@gmail.com>

* Bumo chart to 1.9.0

Signed-off-by: Gerald Barker <gerald.barker@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

cparedes committed:
Add support for customizing service profiles in Spinnaker helm chart. (#10744)

Signed-off-by: Ian Paredes <cp@redbluemagenta.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

raru12 committed:
[stable/minio] create bucket after install through configuration parameter with tls enabled (#10686)

* [stable/minio] update to choose proper connection scheme for connecting minio from mc when creating buckets after install and tls is enabled

Signed-off-by: rahul rustagi <rahulrastogi2k6@gmail.com>

* Added steps in README.md to create buckets after install

Signed-off-by: rahul rustagi <rahulrastogi2k6@gmail.com>
Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
Add mongodb production values priorityClassName comment

Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>

Commit:
Bump mongodb chart version to 5.3.0

Signed-off-by: Victor Boissiere <victor.boissiere@neo9.fr>
